libfreerdp/core/gateway/rts.c

```diff
@@ -857,9 +857,9 @@ static BOOL rts_read_pdu_bind_nak(wStream* s, rpcconn_bind_nak_hdr_t* ctx)
 	WINPR_ASSERT(s);
 	WINPR_ASSERT(ctx);
 
-	if (!Stream_CheckAndLogRequiredLength(
-	        TAG, s, sizeof(rpcconn_bind_nak_hdr_t) - sizeof(rpcconn_common_hdr_t)))
-		return FALSE;
+	// if (!Stream_CheckAndLogRequiredLength(
```

Comment:
There's something to do here

Reply:
That wasn't supposed to be part of the commit...